src/main/java/org/openjdk/shenandoah/ShenandoahVisualizer.java

```diff
@@ -976,6 +990,13 @@ public synchronized void renderRegions(Graphics g) {
                 playback.renderRegions(g);
             }
         }
+        public synchronized void renderLegend(Graphics g) {
```

Comment:
I believe this method is the place that the visualizer draws the legend panel which is the one we tried to change. The difference between `live.renderLegend` and `playback.renderLegend` is that live is for the realtime visualizer and playback is the one that visualizer reads in logfile and generate the data. This method is no longer static is because original legend panel doesn't change at all it is always the same so it could be static. However, now because we want to integrate it with the summary view function, the data will need to be updated all the time. Therefore, I tried to make it seem like the status panel on the top right corner, which will be similar to the `renderStats` method.